public/js/clients/firefox/events.js

```diff
@@ -62,6 +62,11 @@ function newSource(_, packet) {
   if (NEW_SOURCE_IGNORED_URLS.indexOf(source.url) > -1) {
     return;
   }
+
+  if (source.introductionType == "debugger eval") {
```

Comment:
The ignore clause above should do this, yes. Try to figure out why that's not working. We shouldn't need this clause, just fix the `(NEW_SOURCE_IGNORED_URLS` patterns.